indico/util/user.py

```diff
@@ -153,3 +155,25 @@ def wrapper(*args, **kwargs):
         return fn(*args, **kwargs)
 
     return wrapper
+
+
+def create_pending_user(user_data):
+    from indico.modules.users import User
+    from indico.modules.users.models.users import UserTitle
+
+    new_user = User()
+    new_user.email = user_data.get('email').lower()
```

Comment:
this kind of data is required. i'd rather use a separate argument for it or at least use `[]` so it fails if it's missing